tmt/steps/finish/__init__.py

```diff
@@ -104,28 +105,61 @@ def go(self) -> None:
             self.actions()
             return
 
-        # Go and execute each plugin on all guests
+        # Prepare guests
```

Comment:
`Prepare`? I would expect `Finish`

Reply:
This is "prepare" as in "prepare step", it's just a comment, "hey, we're going to prepare some data structures we would use later" :)